src/simulator/halo_simulator.py

```diff
@@ -213,6 +214,46 @@ def simulated_reach_by_spend(
         kplus_reaches = ReachPoint.frequencies_to_kplus_reaches(frequencies)
         return ReachPoint(impressions, kplus_reaches, spends)
 
+    def _liquid_legions_cardinality_estimate_variance(self, n: int) -> float:
+        """Variance of cardinality estimate by unnoised LiquidLegions.
+
+        Args:
+          n:  Carinality of the items contained in a LiquidLegions.
+
+        Returns:
+          var(n-hat) where n-hat is the cardinality estimate from a
+          LiquidLegions which contains n distinct items and has the system
+          parameters (decay rate and sketch size) in this halo.
+        """
+        m = self._params.liquid_legions.sketch_size
+        a = self._params.liquid_legions.decay_rate
+        c = a * n / m / (1 - np.exp(-a))
+        variance = (
+            expi(-c) - expi(-c * np.exp(-a)) - expi(-2 * c) + expi(-2 * c * np.exp(-a))
+        )
+        variance *= a * n ** 2 / m / (np.exp(-c) - np.exp(-c * np.exp(-a))) ** 2
+        variance -= n
+        return variance
+
+    def _liquid_legions_num_active_regions(
```

Comment:
I wonder if the name _liquid_legions_sample_num_active_regions would be more accurate?

Reply:
I believe that num_active_registers is the most commonly used name within Google, and sample_num_active_regions or simply `sampled_regions` also make sense to me.  Now I prefer `num_active_registers`, but is open to other options.